index.html

```diff
@@ -210,16 +209,52 @@ <h2 class="">Multi-Post Stories Gain + Glory</h2>
             </ul>
           </div>
           <div class="flex justify-center">
-            <button class="text-center btn text-white" type="button">
+            <button class="text-center btn text-white btn-view" type="button">
               See Project
             </button>
           </div>
         </div>
       </div>
     </div>
+    <div class="details-modal-overlay"></div>
+    <div class="details-modal">
+      <div class="flex flex-col relative">
+        <div>
+          <img class="detail-img" src="./assets/images/img-placeholder.png" height="500px" alt="imgPlaceholder" />
+          <svg class="absolute btn-close" xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14"
+            fill="none">
+            <path fill-rule="evenodd" clip-rule="evenodd"
+              d="M13.7071 1.70711C14.0976 1.31658 14.0976 0.683417 13.7071 0.292893C13.3166 -0.0976311 12.6834 -0.0976311 12.2929 0.292893L7 5.58579L1.70711 0.292893C1.31658 -0.0976311 0.683417 -0.0976311 0.292893 0.292893C-0.0976311 0.683417 -0.0976311 1.31658 0.292893 1.70711L5.58579 7L0.292893 12.2929C-0.0976311 12.6834 -0.0976311 13.3166 0.292893 13.7071C0.683417 14.0976 1.31658 14.0976 1.70711 13.7071L7 8.41421L12.2929 13.7071C12.6834 14.0976 13.3166 14.0976 13.7071 13.7071C14.0976 13.3166 14.0976 12.6834 13.7071 12.2929L8.41421 7L13.7071 1.70711Z"
+              fill="black" />
+          </svg>
+        </div>
+        <div class="flex justify-between">
+          <h3 class="project-title"></h3>
+          
+        </div>
+        <div class="details-tags">
+          <ul class="flex"></ul>
+        </div>
+        <div class="details-body"></div>
+      </div>
+      <div class="flex justify-between">
+        <button class="detail-btn">
+          <a href="#liveurl" class="flex align-items-center justify-between">
+            <span class="text-white tag-name">See live</span>
+            <img src="./assets/icons/icon-live.svg" alt="live-icon" />
+          </a>
+        </button>
+        <button class="detail-btn">
+          <a href="#github" class="flex align-items-center justify-between">
+            <span class="text-white tag-name">See Source</span>
+            <img src="./assets/icons/icon-github.svg" alt="live-icon" />
+          </a>
+        </button>
+      </div>
+    </div>
   </section>
```

Comment:
- [ ] I kindly suggest that your Modal should follow the Figma design.
- [ ] Kindly add a property of overflow: auto on your `.details-modal` to make sure that the user can see the whole modal.
- [ ] Kindly make sure that your buttons are aligned to the right and not underneath the modal description.

|  | |
| ------------- | ------------- |
| Your Image | ![image](https://github.com/waiyankyaw961999/Microverse-portofolio/assets/52934128/d33f84a3-d369-4434-9b4c-e540f1104e3e)  |
| Figma Design  | ![image](https://github.com/waiyankyaw961999/Microverse-portofolio/assets/52934128/672854f0-55d0-4707-ad2b-26cbc3cea4ef) |